src/fsharp/fsi/Fsi.fsproj

```diff
@@ -31,6 +33,9 @@
     <Compile Include="..\..\assemblyinfo\assemblyinfo.fsi.exe.fs">
       <Link>Utilities/assemblyinfo.fsi.exe.fs</Link>
     </Compile>
+    <Compile Include="..\..\utils\reshaped_reflection.fs">
```

Comment:
Underscores in filenames are non-idiomatic for the codebase.  Use ``ReshapedReflection.fs``?